tests/PSRule.Tests/PSRule.Tests.csproj

```diff
@@ -6,7 +6,7 @@
     <IsTestProject>true</IsTestProject>
     <IsPackable>false</IsPackable>
     <RootNamespace>PSRule</RootNamespace>
-    <DebugType>Full</DebugType>
+    <DebugType>portable</DebugType>
```

Comment:
Change debug type to portable when debugging tests in VSCode, breakpoints weren't getting hit in Full mode. Got this from https://github.com/OmniSharp/omnisharp-vscode/issues/1630.